lib/sportradar/api/nfl/game.rb

```diff
@@ -19,7 +19,11 @@ def initialize(data)
         @utc_offset = data["utc_offset"]
         @weather = data["weather"]
         @clock = data["clock"]
-        @quarter = data["quarter"]
+        if data["quarter"]
+          @quarter = data["quarter"][0]
+          quarter_data = data["quarter"][1].is_a?(Hash) ? [ data["quarter"][1] ] : data["quarter"][1]
+          @quarters = quarter_data&.map { |hash| Sportradar::Api::Nfl::Quarter.new(hash) }
```

Comment:
The current logic is working properly for me, if you see something different, please share it here.